bears/c_languages/GNUIndentBear.py

```diff
@@ -43,7 +43,7 @@ def create_arguments(filename, file, config_file,
                          k_and_r_style: bool=False,
                          linux_style: bool=False,
                          tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
-                         indent_cli_options: str=''):
+                         cli_options: str=''):
```

Comment:
I'm against this, I think we should keep the tool name in the setting for additional options because it's tool specific